src/v3/src/components/Widget/index.tsx

```diff
@@ -493,27 +494,29 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     >
       <CustomPluginsOdysseyCacheProvider nonce={cspNonce}>
         <MuiThemeProvider theme={theme}>
```

Comment:
Can you put a comment here indicating that we can remove this provider when we remove all the `-legacy` imports?